contracts/VetoGuard.sol

```diff
@@ -83,8 +91,11 @@ contract VetoGuard is
         );
 
         require(
-            transactionQueuedBlock[transactionHash] == 0,
-            "Transaction has already been queued"
+            block.timestamp >=
+                transactionQueuedTimestamp[transactionHash] +
+                    timelockPeriod +
+                    executionPeriod,
+            "Transaction has already been queued recently"
```

Comment:
why was "recently" added to the end here?

Reply:
I added that because the logic there changed - The conditional that was removed there prevented the same tx from ever getting queued twice. However it should be possible to queue up a tx again if it has surpassed its execution period. This is what that new conditional allows for

Reply:
ah ok, that makes sense